PurchasesCoreSwift/StoreKitExtensions/SKPaymentTransaction+Extensions.swift

```diff
@@ -13,13 +13,12 @@
 
 import StoreKit
 
-// todo: make internal after migration
-public extension SKPaymentTransaction {
+extension SKPaymentTransaction {
 
     /// Considering issue https://github.com/RevenueCat/purchases-ios/issues/279, sometimes `payment`
     /// and `productIdentifier` can be `nil`, in this case, they must be have treated as nullable.
     /// Due of that an optional reference is created so that the compiler would allow us to check for nullability.
-    @objc var rc_productIdentifier: String? {
+    var rc_productIdentifier: String? {
```

Comment:
if this is internal, can we remove the rc prefix? not sure what convention we've defined for that...

Reply:
we don't have a convention for it I believe, although I've seen a few files where if the extension is `private`, there's no prefix, and if it's `internal` or above, the prefix is added. 
I'd be fine with having it be "only add the prefix if public", since that way we're ensuring that we don't cause name conflicts.